docs/output.md

```diff
@@ -4,74 +4,117 @@
 
 This page describes the output produced by the pipeline.
 
-The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level output directory (specified by `--outdir <DIR>`). This is an example output when the pipeline has been run for a WGS sample called `SAMPLE_1` and a WES sample called `SAMPLE_2` which form a family called `FAMILY_1`. The output consists of 4 directories: `yyyy-MM-dd_project_name`, `individuals`, `multiqc_reports` and `pipeline_info`. This run has only been run with `haplotypecaller`: (`--callers haplotypecaller`)
+The output directory has been structured in such a way that you can pass the same output directory to it for each pipeline run. The pipeline will add the files to that directory in a traceable way without overwriting already existing files. This makes it easy to store data, coming from multiple sequencing runs, in the same root directory.
+
+To explain the structure of the output directory, a simple example run consisting of two families is used. The first family (`family1`) is a family consisting of a trio (son, father and mother) and the second family (`family2`) consists of a single sample.
 
 ```bash
-results/
-├── YYYY_MM_DD_project_name #(1)!
-│   └── FAMILY_1 #(2)!
-│       ├── FAMILY_1.bed #(3)!
-│       ├── FAMILY_1.haplotypecaller.ped #(4)!
-│       ├── FAMILY_1.haplotypecaller.vcf.gz #(5)!
-│       ├── FAMILY_1.haplotypecaller.vcf.gz.tbi #(6)!
-│       └── reports
-│           ├── FAMILY_1.haplotypecaller.bcftools_stats.txt #(7)!
-│           └── FAMILY_1.haplotypecaller.somalier.html #(8)!
-├── multiqc
-│   ├── multiqc_data/
-│   └── multiqc_report.html #(9)!
-├── SAMPLE_1 #(10)!
-│   ├── SAMPLE_1.bed #(11)!
-│   ├── SAMPLE_1.haplotypecaller.g.vcf.gz #(12)!
-│   ├── SAMPLE_1.haplotypecaller.g.vcf.gz.tbi
-│   └── reports
-│       ├── SAMPLE_1.haplotypecaller.bcftools_stats.txt
-│       ├── SAMPLE_1.mosdepth.global.dist.txt #(13)!
-│       └── SAMPLE_1.mosdepth.summary.txt #(14)!
-├── SAMPLE_2
-│   ├── SAMPLE_2.bed
-│   ├── SAMPLE_2.haplotypecaller.g.vcf.gz
-│   ├── SAMPLE_2.haplotypecaller.g.vcf.gz.tbi
-│   └── reports
-│       ├── SAMPLE_2.haplotypecaller.bcftools_stats.txt
-│       ├── SAMPLE_2.mosdepth.global.dist.txt
-│       └── SAMPLE_2.mosdepth.summary.txt
-├── pipeline_info/ #(15)!
-└── samplesheet.csv #(16)!
+<outdir> #(1)!
+├── family1 #(2)!
+│   ├── output_<pipeline_version>_<date> #(3)!
+│   │   ├── automap #(4)!
+│   │   │   └── <caller> #(5)!
```

Comment:
I think this is more clear because it only contains the automap files. You see `automap` first and then the `caller` if you view it in a file explorer